### ISSUE TITLE
Mention cordova shortcomings/known issues

### ISSUE THREAD

**Issue by nickve28** [kabisa/maji#248]
It would be nice if we could describe known issue we've encountered in cordova, like websql/indexeddb being cleared.

This is never mentioned anywhere in the cordova docs as far as I've found https://cordova.apache.org/docs/en/latest/cordova/storage/storage.html

**Comment by pascalw** [comment]
@nickve28 good point. As we discussed recently we should ship `nativeStorage` support out of the box, so this is no longer an issue at all.
